rules/directory-traversal.go

```diff
@@ -0,0 +1,32 @@
+package rules
+
+import (
+	"go/ast"
+
+	"github.com/securego/gosec/v2"
+)
+
+type traversal struct {
+	gosec.MetaData
+}
+
+func (r *traversal) ID() string {
+	return r.MetaData.ID
+}
+
+func (r *traversal) Match(n ast.Node, ctx *gosec.Context) (*gosec.Issue, error) {
+	return gosec.NewIssue(ctx, n, r.ID(), r.What, r.Severity, r.Confidence), nil
```

Comment:
Are you planning to implement this rule?

Reply:
Yes, this is currently just a draft, will get it working in the upcoming days.